auto_nag/scripts/unaffected_affected_no_reg.py

```diff
@@ -30,6 +30,8 @@ def get_bz_params(self, date):
         release_version, beta_version, central_version = getVersions()
         params = {'keywords': ['regression', 'feature'],
                   'keywords_type': 'nowords',
+                  'short_desc_type': 'notregexp',
+                  'short_desc': '.*([pP]ermafailing|[pP]erma-failing|[pP]erma |[iI]ntermittent |[bB]uild bustage|[bB]ustages on).*',
```

Comment:
maybe we could add this regex in config file like we did for regression::product_blacklist (https://github.com/mozilla/relman-auto-nag/blob/master/auto_nag/scripts/configs/tools.json#L45).
wdyt ?